src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -686,10 +688,10 @@ void OpenGLDrawingContext::DrawSprite(uint32_t image, int32_t x, int32_t y, uint
     right -= _dpi->x;
     bottom -= _dpi->y;
 
-    left /= zoomLevel;
-    top /= zoomLevel;
-    right /= zoomLevel;
-    bottom /= zoomLevel;
+    left = left / _dpi->zoom_level;
```

Comment:
Why this change?

Reply:
You can't add new `/=` operators for types like `int`.